src/middlewares/authorization.js

```diff
@@ -0,0 +1,17 @@
+import verifyToken from '../utils/authentication';
+
+const authentication = (req, res, next) => {
+  const bearerHeader = req.headers['x-access-token'] || req.query.token || req.headers.authorization;
+  if (bearerHeader) {
+    const bearer = bearerHeader.split(' ');
+    const token = bearer[1];
+    verifyToken(req, res, token);
```

Comment:
We don't want to pass the req n res object to the `verifyToken` method. see my comment on the `src/utils/authentication.js`file below for more clarification